include/ipc.h

```diff
@@ -27,8 +28,10 @@ enum ipc_command_type {
 	IPC_EVENT_WINDOW = ((1<<31) | 3),
 	IPC_EVENT_BARCONFIG_UPDATE = ((1<<31) | 4),
 	IPC_EVENT_BINDING = ((1<<31) | 5),
-	IPC_EVENT_MODIFIER = ((1<<31) | 6),
-	IPC_EVENT_INPUT = ((1<<31) | 7),
+	IPC_EVENT_SHUTDOWN = ((1<<31) | 6),
+	IPC_EVENT_TICK = ((1<<31) | 7),
+	IPC_EVENT_MODIFIER = ((1<<31) | 16),
+	IPC_EVENT_INPUT = ((1<<31) | 17),
```

Comment:
Why 16 and 17?

Sway 0.15 uses the following - not sure if we need to use the same:

    IPC_EVENT_MODIFIER = ((1<<31) | 6),
    IPC_EVENT_INPUT = ((1<<31) | 7),

(`SHUTDOWN` and `TICK` aren't listed in 0.15. I guess there's no events for those.)

Reply:
We need to match the i3 dudes where they line up and we need to put our extensions far away from them so i3 has room to grow.

Reply:
If the extensions have to be far away, what about using ```((1<<31) | (1<<30) | 1)``` and higher for sway specific events?

Reply:
I don't think SirCmpwn noticed the `1<<31`. These values are already far away from i3's.

My point is sway 0.15 already has `(1<<31) | 6` allocated to `IPC_EVENT_MODIFIER`. My comment is about being compatible with sway 0.15, not with i3.

Reply:
I think this is a moot point anyway because as far as I can tell, `IPC_EVENT_MODIFIER` and `IPC_EVENT_INPUT` don't actually appear anywhere else in the code.

Reply:
Eh, modifier events are actually used in 0.15, but not input events. So I think it depends how important modifier events are to be preserved.

Anyway, I'm more inclined to bring the values in line with i3.

Reply:
@RyanDwyer (1<<31) is not sway specific. This bit marks an event sent from i3/sway to the client after the client did IPC_SUBSCRIBE.

Reply:
Any further thoughts on:
1. removing or implementing `IPC_EVENT_INPUT`;
2. removing or implementing `IPC_EVENT_MODIFIER`?

Reply:
I would think that these events need to be preserved/implemented, unless there's a protocol which does their job. Ping @SirCmpwn

Reply:
We should keep `IPC_EVENT_INPUT` around. Does anyone know what `IPC_EVENT_MODIFIER` is used for? Also, let's assign these to '31' and '30', no need to be shy.

Reply:
I'm actually thinking 100+ to match the non-event sway-specific values. Also, it seems that `IPC_EVENT_MODIFIER` is used to indicate when a bar modifier is pressed/released (which doesn't seem to be implemented in 1.0 yet...)

Reply:
Let's drop `IPC_EVENT_MODIFIER`.

Reply:
Ideas on what to use `IPC_EVENT_INPUT` for? Maybe I can implement it. Or leave it for now?

Reply:
Actually, no, I was confusing it with `IPC_GET_INPUTS` which we should keep. We can drop `IPC_EVENT_INPUT` too.